{{cookiecutter.project_slug}}/.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,149 @@
+---
+name: 🐛  Bug Report
+description: Create a report to help us improve
+labels: ["bug", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug in {{ cookiecutter.project_name }}!**
```

Comment:
@ayeankit did you check if `{{ }}` are working fine?

Reply:
didn't test it .. but it should work fine in theory

Reply:
I didn't check locally but as per as documentation it should work fine.